health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/CampaignUpdateSummary.js

```
@@ -177,39 +178,55 @@ const CampaignUpdateSummary = (props) => {
-                    },
-                  },
-                ],
-              };
-            }),
+                    sections: [
+                      {
+                        name: `HIERARCHY_${index + 1}`,
+                        type: "COMPONENT",
+                        cardHeader: { 
+                          value: hierarchyType 
+                              ? `${t((hierarchyType + "_" + item?.type).toUpperCase())}` 
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)